pages/index.js

```diff
@@ -1,12 +1,39 @@
 import Head from 'next/head'
-import Layout from '../components/layout'
-import { LoginButton} from '../components/auth'
+import { useUser } from "@auth0/nextjs-auth0";
+import { useRouter } from 'next/router';
 
+import { Disclosure } from "@headlessui/react";
+import { ChevronDownIcon } from "@heroicons/react/outline";
+import clsx from "clsx";
 
-export default function Feed() {
-  return (
-    <div>TODO (maybe redirect to /feed if logged in)
-      <LoginButton/>
+
+
+export default function Index() {
+  const { user, error, isLoading } = useUser();
+  const router = useRouter();
+
+  if (isLoading) return <div>Loading...</div>;
+  if (error) return <div>{error.message}</div>;
+
+  if (user) {
+    router.push("/feed");
+  }
```

Comment:
It might be worthwhile to have the homepage/feed both be on root -> thinking of how EVERYONE ELSE does it.

Reply:
Yeah, I agree with that.